features/FEATURE_LWIP/lwip-interface/lwip/FILES

```diff
@@ -1,4 +1,5 @@
 src/      - The source code for the lwIP TCP/IP stack.
 doc/      - The documentation for lwIP.
+test/     - Some code to test whether the sources do what they should.
```

Comment:
Just lwip's internal tests. The directory has been there all the time but was missing from list.